js/binance.js

```diff
@@ -710,6 +755,7 @@ module.exports = class binance extends Exchange {
                         },
                     },
                 },
+                'option': {},
```

Comment:
This doesn't work

Reply:
I don't really understand what you mean when you say _"doesn't work"_. What is expected, what do you get instead, can you fix/resolve it ?

Reply:
I mean, see how it's done here for futures: https://github.com/ccxt/ccxt/blob/master/js/binance.js#L1546 < for we should copy the same approach for options, i think.